style.css

```diff
@@ -0,0 +1,123 @@
+
+.container {
+  display: flex;
+  position: absolute;
+  width: 1280px;
+  height: 768px;
+  background-image: url("/imgs/Header\ bg.svg");
+  align-content: center;
+  padding: 200px;
+}
+
+.Portfolio {
+  width: 60px;
+  height: 20px;
+  font-style: normal;
```

Comment:
[ ] kindly use the same `font-family` used in the Figma template, wich is `Poppins` 🤓